srcStatic/GameModules/ConsoleModule.cpp

```diff
@@ -0,0 +1,63 @@
+#include "ConsoleModule.h"
+#include "../FileSystemHelper.h"
+#include "../Log.h"
+#include <windows.h>
+#include <stdexcept>
+
+ConsoleModule::ConsoleModule(const std::string& moduleName) : DllModule(moduleName)
+{
+	moduleDirectory = (fs::path(GetGameDirectory()) / moduleName).string() + "\\";
+
+	std::error_code errorCode;
+	if (!fs::is_directory(moduleDirectory, errorCode)) {
```

Comment:
This is likely to create problems for Mingw. The `is_directory` method will return false for any paths with a trailing slash. It maybe be easier to append the slash after checking `is_directory`.

Alternatively, perhaps we should create an `IsDirectory` method in FileSystemHelper, which contains logic to work around Mingw bugs.